src/gettingstarted/local/docksal.md

```diff
@@ -0,0 +1,72 @@
+# Using Docksal for local development
+
+[Docksal](https://docksal.io) is a docker based local development tool that plays nicely with Platform.sh.  It is maintained by a community of developers but is a viable option for most Platform.sh projects.
+
+See the [Docksal documentation](https://docs.docksal.io/) for installing and setting up Docksal on your system.
+
+Docksal will ask you to create a `.docksal` directory in your application root, which functions similarly to the `.platform.app.yaml` file. It is safe to check this directory into your Git repository as Platform.sh will simply ignore it.
+
+All interaction with a Docksal project will happen through the `fin` command. For more info you can type the `help` subcommand to get all available commands and options.
+
+```bash
+fin help
+```
+
+## Using Platform.sh CLI within Docksal
+
+To start using the Platform.sh CLI within your docksal project. The `SECRET_PLATFORMSH_CLI_TOKEN` will need to be set. This will allow for you to interact with your Platform.sh account without the need to download anything locally.
```

Comment:
This should also specify what the token is and where to get it. It's best to just call it an API token and link to the existing CPI/API Tokens page.

Reply:
Completely understand. Is there a standard URL I can use as the one I would normally link to has an account id?

Reply:
Yep, link with Markdown syntax to: /gettingstarted/cli/api-tokens.md.  (The GitBook builder will convert it to an HTML link for us.)